argus/action_manager/windows.py

```diff
@@ -425,11 +425,35 @@ def __init__(self, client, os_type=util.WINDOWS8):
         super(Windows8ActionManager, self).__init__(client, os_type)
 
 
+class WindowsServer2008ActionManager(WindowsActionManager):
+    def __init__(self, client, os_type=util.WINDOWS_SERVER_2008):
+        super(WindowsServer2008ActionManager, self).__init__(client, os_type)
+
+
+class WindowsServer2008R2ActionManager(WindowsActionManager):
```

Comment:
afaik, there is no difference on the API on 2008/2008R2, so there is no need to have an action manager for each